src/handlers/use_handler.rs

```diff
@@ -313,7 +313,7 @@ fn add_to_path(installation_dir: &Path) -> Result<()> {
             let env = current_usr.open_subkey_with_flags("Environment", KEY_READ | KEY_WRITE)?;
             let usr_path: String = env.get_value("Path")?;
 
-            if usr_path.contains(installation_dir) {
+            if usr_path.to_lowercase().replace('/', "\\").contains(&installation_dir.to_lowercase()) {
```

Comment:
separate the lowercase ones to variables instead, looks a bit ugly.

Reply:
The last commit should hopefully address both comments